sysfs/vulnerability.go

```diff
@@ -59,29 +73,37 @@ func (fs FS) CPUVulnerabilities() ([]Vulnerability, error) {
 // Vulnerability represents a single vulnerability extracted from /sys/devices/system/cpu/vulnerabilities/.
 type Vulnerability struct {
 	CodeName   string
-	State      string
+	State      int
 	Mitigation string
 }
 
-func parseVulnerability(name, value string) (Vulnerability, error) {
-	v := Vulnerability{CodeName: name}
-	value = strings.TrimSpace(value)
-	if value == notAffected {
-		v.State = notAffected
+func parseVulnerability(name, rawContent string) (*Vulnerability, error) {
+	v := &Vulnerability{CodeName: name}
+	rawContent = strings.TrimSpace(rawContent)
+	rawContentLower := strings.ToLower(rawContent)
```

Comment:
Maybe simplify and just use the lower string for everything.

```suggestion
	vulnerabilityState := strings.ToLower(strings.TrimSpace(rawContent))
```

Reply:
@SuperQ 

I didn't want to lose upper cases in the mitigation text, for example:
```
Retpolines, IBPB: conditional, STIBP: always-on, RSB filling, PBRSB-eIBRS: Not affected
```

It felt like bringing the whole text to lower cases would result in reduced legibility. I lean towards leaving cases as they are, but I don't feel particularly strongly about it. If you think that simplifying this to a single var is more important than keeping the cases than let me know and I'll push another patch.

Reply:
Sure, that makes sense to keep the original string caps.